grovedb/Cargo.toml

```
@@ -50,8 +51,9 @@ name = "insertion_benchmark"
 harness = false
 
 [features]
-default = ["full"]
+default = ["full", "serde-support"]
 proof_debug = ["grovedb-merk/proof_debug"]
+serde-support = ["serde", "grovedb-merk/serde-support", "indexmap/serde"]
```

I think `serde` is a de facto standard for the feature name

```suggestion
serde = ["dep:serde", "grovedb-merk/serde-support", "indexmap/serde"]
```

---

doen